src/app/member/components/MemberHeader.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { it, describe, vi } from "vitest";
+import { MemeberHeader } from "./MemeberHeader";
+
+describe("Header Component", () => {
+  beforeAll(() => {
```

Comment:
I think we should not need these props here

Reply:
@Mubashar1009 same here